G/GAP_lib/build_tarballs.jl

```diff
@@ -22,7 +22,7 @@ using BinaryBuilder, Pkg
 
 name = "GAP_lib"
 upstream_version = v"4.14.0"
-version = v"400.1400.002"
+version = v"400.1401.000"
```

Comment:
```suggestion
version = v"400.1400.003"
```
I am not sure which of the two versions better matches the GAP_lib_jll versioning scheme. @fingolfin please decide and either apply the patch or don't.

Reply:
Either is fine by me, as this change is fully backwards compatible. For more risky changes I'd make sure to increment the minor version to ensure nobody gets the changes without also updating GAP.jl. Here either should be fine...

That said I guess the slightly safer approach is to use 1401 as you do, again to ensure nobody out there gets these changes before we make a new GAP.jl release